spec/services/get_application_choices_ready_to_decline_by_default_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe GetApplicationChoicesReadyToDeclineByDefault do
+  around do |example|
+    Timecop.freeze(Time.zone.local(2019, 12, 4, 12, 0, 0)) do
+      example.run
+    end
+  end
+
+  let(:application_form) { create :application_form }
+
+  def create_application(status:, decline_by_default_at:)
+    create(
+      :application_choice,
+      application_form: application_form,
+      status: status,
+      decline_by_default_at: decline_by_default_at,
+    )
+  end
+
+  it 'returns application choices with decline_by_default_at in the past' do
+    create_application(
+      status: 'offer',
+      decline_by_default_at: 1.business_days.ago,
+    )
+    create_application(
+      status: 'offer',
+      decline_by_default_at: 2.business_days.from_now,
+    )
+    create_application(
+      status: 'offer',
+      decline_by_default_at: 3.business_days.ago,
+    )
+    Timecop.travel(1.business_days.from_now) do
+      choices = described_class.call
+      expect(choices).to include application_form.application_choices.first
```

Comment:
This would have been slightly clearer to me if you stored the return values from the calls to `create_application`, e.g. `application_choice1 = create_application(...` then just made the assertions `expect(choices).to include application_choice1` etc. (very minor and debateable)

Reply:
Was going to say debateable but it does actually make it easier to read.